apps/remix-ide-e2e/src/tests/proxy.test.ts

```diff
@@ -0,0 +1,289 @@
+      .click('[data-id="confirmProxyDeployment-modal-footer-ok-react"]')
+      .waitForElementPresent('[data-id="universalDappUiTitleExpander0"]')
+      .waitForElementPresent('[data-id="universalDappUiTitleExpander1"]')
+  },
+
+  'Should interact with deployed contract via ERC1967 (proxy)': function (browser: NightwatchBrowser) {
+    browser
+      .getAddressAtPosition(1, (address) => {
+        console.log('Proxy address: ' + address)
+        firstProxyAddress = address
+      })
+      .clickInstance(1)
+      .perform((done) => {
+        browser.testConstantFunction(firstProxyAddress, 'name - call', null, '0:\nstring: MyToken').perform(() => {
+          done()
+        })
+      })
+      .perform((done) => {
+        browser.testConstantFunction(firstProxyAddress, 'symbol - call', null, '0:\nstring: MTK').perform(() => {
+          done()
+        })
+      })
+  },
+
+  'Should deploy proxy with initialize parameters': function (browser: NightwatchBrowser) {
+    browser
+      .waitForElementPresent('[data-id="deployAndRunClearInstances"]')
+      .click('[data-id="deployAndRunClearInstances"]')
+      .addFile('initializeProxy.sol', sources[2]['initializeProxy.sol'])
+      .clickLaunchIcon('solidity')
+      .pause(2000)
+      .click('[data-id="compilerContainerCompileBtn"]')
+      .waitForElementPresent('select[id="compiledContracts"] option[value=MyInitializedToken]', 60000)
+      .clickLaunchIcon('udapp')
+      .click('select.udapp_contractNames')
+      .click('select.udapp_contractNames option[value=MyInitializedToken]')
+      .waitForElementPresent('[data-id="contractGUIDeployWithProxyLabel"]')
+      .click('[data-id="contractGUIDeployWithProxyLabel"]')
+      .waitForElementPresent('input[title="tokenName"]')
+      .waitForElementPresent('input[title="tokenSymbol"]')
+      .setValue('input[title="tokenName"]', 'Remix')
+      .setValue('input[title="tokenSymbol"]', "R")
+      .createContract('')
+      .waitForElementContainsText('[data-id="udappNotifyModalDialogModalTitle-react"]', 'Deploy Implementation & Proxy (ERC1967)')
+      .waitForElementVisible('[data-id="udappNotify-modal-footer-ok-react"]')
+      .click('[data-id="udappNotify-modal-footer-ok-react"]')
+      .waitForElementContainsText('[data-id="confirmProxyDeploymentModalDialogModalTitle-react"]', 'Confirm Deploy Proxy (ERC1967)')
+      .waitForElementVisible('[data-id="confirmProxyDeployment-modal-footer-ok-react"]')
+      .click('[data-id="confirmProxyDeployment-modal-footer-ok-react"]')
+      .waitForElementPresent('[data-id="universalDappUiTitleExpander0"]')
+      .waitForElementPresent('[data-id="universalDappUiTitleExpander1"]')
+  },
+
+  'Should interact with initialized contract to verify parameters': function (browser: NightwatchBrowser) {
+    browser
+      .getAddressAtPosition(1, (address) => {
+        console.log('Proxy address: ' + address)
```

Comment:
same here

Reply:
Console logs should be left for easy debugging in circleci.